packages/http/src/mocker/negotiator/InternalHelpers.ts

```diff
@@ -49,7 +49,7 @@ export function findDefaultContentType(
 
 const byResponseCode = ord.contramap<number, IHttpOperationResponse>(ordNumber, response => parseInt(response.code));
 
-export function findLowest2xx(httpResponses: NonEmptyArray<IHttpOperationResponse>): Option<IHttpOperationResponse> {
+export function findLowest2xx(httpResponses: IHttpOperationResponse[]): Option<IHttpOperationResponse> {
```

Comment:
`NonEmptyArray`, I'll miss U 💔

Reply:
Yeah I really thought this was something cool.